react/features/chat/middleware.js

```diff
@@ -94,15 +95,18 @@ function _addChatMsgListener(conference, { dispatch, getState }) {
             // backfilled for a participant that has left the conference.
             const participant = getParticipantById(state, id) || {};
             const displayName = participant.name
-                || `${interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME} (${id})`;
+                || (typeof interfaceConfig === 'object'
```

Comment:
I recall this being pretty old logic. I think it would be okay to explore changing things a bit if necessary instead of holding onto old web logic.

If no changes are going to be made though, does the case of no participant.name and interfaceconfig being undefined need to be handled? Currently that case would set display name to false.

Reply:
That's what `getParticipantDisplayName` solves. If falls back to a default, if the participant is lacking one.